### PR TITLE
Update the`vip validate` command group to follow the VIP-CLI style guide

### PR DESCRIPTION
**Pull request by yolih**
## Description

Updated the voice, tense, and format of command descriptions, examples, and usage models for the `vip validate` command group to follow the VIP-CLI style guide.

## Pull request checklist

- [ n/a ] Update [SETUP.md](https://github.com/Automattic/vip-cli/blob/trunk/docs/SETUP.md#list-of-environmental-variables) with any new environmental variables.
- [ ] Update [the documentation](https://github.com/Automattic/vip-cli/blob/trunk/docs).
- [ ] [Manually test](https://github.com/Automattic/vip-cli/blob/trunk/docs/TESTING.md#manual-testing) the relevant changes.
- [ x ] Follow the [pull request checklist](https://github.com/Automattic/vip-cli/blob/trunk/docs/RELEASING.md#new-pull-requests)
- [ x ] Add/update [automated tests](https://github.com/Automattic/vip-cli/blob/trunk/docs/TESTING.md#automated-testing) as needed.

## New release checklist

- [ ] [Automated tests](https://github.com/Automattic/vip-cli/blob/trunk/docs/TESTING.md#automated-testing) pass.
- [ ] The [Preparing for release checklist](https://github.com/Automattic/vip-cli/blob/trunk/docs/RELEASING.md#preparing-for-release) is completed.

## Steps to Test

Observe the results of the `--help` display for impacted commands:

- `vip validate`
- `vip validate preflight`

### CODE REVIEW

File: src/bin/vip-logs.js [hunk 1]
```diff
@@ -243,7 +243,7 @@ command( {
 		`The maximum number of entries to return. Accepts an integer value between 1 and 5000 (defaults to ${ LIMIT_DEFAULT }).`
 	)
 	.option( 'follow', 'Output new entries as they are generated.' )
-	.option( 'format', 'Render output in a particular format. Accepts “csv”, and “json”.', 'table' )
+	.option( 'format', 'Render output in a particular format. Accepts “csv” and “json”.', 'table' )
```
Comment (sjinks) [inline src/bin/vip-logs.js]: It looks like `table` is a valid value as well.

File: src/bin/vip-slowlogs.ts [hunk 2]
```diff
@@ -199,7 +199,7 @@ void command( {
 		'Set the maximum number of log entries. Accepts an integer value between 1 and 500.',
 		500
 	)
-	.option( 'format', 'Render output in a particular format. Accepts “csv”, and “json”.', 'table' )
+	.option( 'format', 'Render output in a particular format. Accepts “csv” and “json”.', 'table' )
```
Comment (sjinks) [inline src/bin/vip-slowlogs.ts]: Same - `table` is a valid value as well.

File: src/bin/vip-validate-preflight.js [hunk 3]
```diff
@@ -576,43 +578,58 @@ if ( parsedAlias.app ) {
...
 		3000
 	)
 	.option(
 		[ 'p', 'port' ],
-		'Configure the port to use for the app (defaults to a random port between 3001 and 3999)'
+		'Set a port for the application. (Defaults to a random value between 3001 and 3999)'
 	)
-	.option( 'format', 'Output the log lines in CSV or JSON format', 'table' )
-	.option( [ 'P', 'path' ], 'Path to the app to be tested', process.cwd() )
+	.option( 'format', 'Render output in a particular format. Accepts “csv” and “json”.', 'table' )
```
Comment (sjinks) [inline src/bin/vip-validate-preflight.js]: Same

File: src/bin/vip-validate-preflight.js [hunk 4]
```diff
@@ -577,42 +579,57 @@ if ( parsedAlias.app ) {
...
 }
 
-command( commandOpts )
+const usage = 'vip validate preflight';
+
+command( { commandOpts, usage } )
 	.option(
 		'verbose',
-		'Increase logging level to include app build and server boot up messages',
+		'Increase logging level to include app build and server boot up messages. Outputs rows to the console as they are generated.',
```
Comment (rinatkhaziev) [inline src/bin/vip-validate-preflight.js]: How's it relevant? I.e. why do we need to call this out?

File: src/bin/vip-validate-preflight.js [hunk 5]
```diff
@@ -577,42 +579,57 @@ if ( parsedAlias.app ) {
...
 		false
 	)
 	.option(
 		'node-version',
-		`Select a specific target Node.JS version in semver format (MAJOR.MINOR.PATCH) or a MAJOR (${ ALLOWED_NODEJS_VERSIONS.join(
+		`Set a version of Node.JS for the tests. Accepts semver format (MAJOR.MINOR.PATCH) or a MAJOR (${ ALLOWED_NODEJS_VERSIONS.join(
```
Comment (rinatkhaziev) [inline src/bin/vip-validate-preflight.js]:
```suggestion
		`Set a version of Node.js for the tests. Accepts semver format (MAJOR.MINOR.PATCH) or a MAJOR (${ ALLOWED_NODEJS_VERSIONS.join(
```

**Review by rinatkhaziev — APPROVED** [review]

File: src/bin/vip-validate-preflight.js (hunk 4 above)
Comment (yolih) [reply src/bin/vip-validate-preflight.js]: Agreed! Will remove.

File: src/bin/vip-validate-preflight.js (hunk 5 above)
Comment (yolih) [reply src/bin/vip-validate-preflight.js]: Whoopsie. Thank you - fixing!

**Pull request merged**